src/features/analytics/types.ts

```diff
@@ -0,0 +1,61 @@
+import { TokenStandard } from '@hyperlane-xyz/sdk';
+
+import { ProtocolType } from '@hyperlane-xyz/utils';
+
+export enum EVENT_NAME {
+  CHAIN_SELECTION = 'Chain Selection',
+  TOKEN_SELECTION = 'Token Selection',
+  TRANSACTION_SUBMISSION = 'Transaction Submission',
+  TRANSACTION_SUBMISSION_FAILED = 'Transaction Submission Failed',
+  WALLET_CONNECTION_INITIATED = 'Wallet Connection Initiated',
+  WALLET_CONNECTED = 'Wallet Connected',
+}
```

Comment:
Based on the doc you shared aren't we missing some events here? like the msg dispatched and processed?